ts/features/pn/components/PnMessageDetails.tsx

```diff
@@ -114,6 +114,11 @@ export const PnMessageDetails = (props: Props) => {
 
   const scrollViewRef = React.createRef<ScrollView>();
 
+  const errorOrUndefined = paymentVerificationError.toUndefined();
+  const isPaid =
+    errorOrUndefined === "PAA_PAGAMENTO_DUPLICATO" ||
```

Comment:
What do you think about managing this check in just one place, like:
```ts
function isDuplicatedPayment(detail: Detail_v2Enum | undefined): boolean {
  return (
    detail === "PAA_PAGAMENTO_DUPLICATO" || detail === "PPT_PAGAMENTO_DUPLICATO"
  );
}
```
And then reusing this function everywhere's needed? 🤔

Reply:
@dgopsq I haven't done it because the error type is a bit more complex than that... however it makes absolutely sense 0e5759f324e222f1421b368ed950e18002261d5a